src/views/postUpdation/AllPostTypesUpdation.tsx

```diff
@@ -444,6 +478,34 @@ export const AllPostTypesUpdation: React.FC<IAllPostTypesUpdation> = ({
         disabled={isDisabled}
         post={post}
       />
+      <BorderBottom />
+      <Typography component="h2" variant="h2">
+        {t(langTokens.admin.changesList)}
+      </Typography>
+      <Box mt={3}>
+        <AuthorListDropdown
+          pageSizes={changesPerPage}
+          setChanges={handleNotesToShowChange}
+          selected={size || changesPerPage[changesPerPage.length - 1]}
```

Comment:
if we update `IAuthorListDropdownProps` to allow only `All` among other strings, we could also hard-code it here:

```
selected={size || 'All'}
```
and TS will keep us safe